src/store/index.ts

```diff
@@ -2,14 +2,15 @@ import { createStore, applyMiddleware } from 'redux';
 import thunk, { ThunkDispatch } from 'redux-thunk';
 import rootReducer from './reducers';
 import { AppActions } from '../actions';
+import recordAction from './recordAction';
```

Comment:
I'm thinking ahead to deploying this as a stand-alone NPM package that would live at `@fullstory/record-redux-action`